### DISCUSSION TITLE
Add to dictionary in Nextcloud Office

### DISCUSSION THREAD

**Discussion by karlemilnikka** [nextcloud/all-in-one#1139] [Questions] [answered]
Has anyone got “add to dictionary” to work in Nextcloud Office, running on Nextcloud AIO? The option is missing when right-clicking on words that are indicated to be misspelled.

**Comment by szaimen** [comment]
I haven't tried but I do not see why it should behave differently then a normal collabora container since it is basically the official one...

So I would suggest reporting this to upstream collabora or asking the question there.

**Comment by karlemilnikka** [comment]
Thanks @szaimen. 

New post at Collabora: https://github.com/CollaboraOnline/online/discussions/5238.

**Comment by karlemilnikka** [comment, marked as answer]
A Collabora maintainer has answered that the feature is intentionally removed. https://github.com/CollaboraOnline/online/discussions/5238

**Comment by P4z** [comment]
@karlemilnikka, another developer also answered in https://github.com/CollaboraOnline/online/discussions/5238#discussioncomment-10412769 :

> These words should be saved in the integrating environment (eg. Nextcloud), the feature would need to implemented on that end, and as an interface to Collabora Online. There are also tricky situations, because originally, in LibreOffice these additions are saved to the user profile, but in this case not only a single user can work on a document, so storing these per user on the file host might not make much sense.

Could we expect any progress on your side? Being able to add new words to a custom dictionary is a core feature when editing documents to me.

**Comment by karlemilnikka** [comment]
@P4z I’m not part of the Collabora or Nextcloud team, so I have to redirect the question to @szaimen.

**Reply by P4z** [reply]
Thank you. I'm sorry for the mistake.

**Reply by szaimen** [reply]
This needs to be implemented in collabora afaics. So the linked thread is probably correct

**Reply by karlemilnikka** [reply]
@szaimen Do you think the new Euro-office based Nextcloud Office will get support for adding words to dictionaries (when running on Nextcloud AIO)?

**Reply by szaimen** [reply]
Not sure. Best to ask this in an appropriate place e.g. here: https://github.com/Euro-Office/eurooffice-nextcloud/issues
